ament_package/template/isolated_prefix_level/local_setup.bat.in

```diff
@@ -24,14 +24,23 @@ goto:eof
   setlocal enabledelayedexpansion
 
   :: use the Python executable known at configure time
-  set "ament_python_executable=@PYTHON_EXECUTABLE@"
+  set "_ament_python_executable=@PYTHON_EXECUTABLE@"
   :: allow overriding it with a custom location
   if "%AMENT_PYTHON_EXECUTABLE%" NEQ "" (
-    set "ament_python_executable=%AMENT_PYTHON_EXECUTABLE%"
+    set "_ament_python_executable=%AMENT_PYTHON_EXECUTABLE%"
+  )
+  if NOT exist %_ament_python_executable% (
+    for %%I in (python.exe) do (
+      if "%%~$PATH:I" NEQ "" set "_ament_python_executable=%%~$PATH:I"
+    )
+  )
+  if NOT exist %_ament_python_executable% (
+    echo Could not find Python executable: either add the folder of the Python executable (e.g. installed with Chocolatey) to the PATH variable or explicitly set the environment variable AMENT_PYTHON_EXECUTABLE to the full path of the Python executable
```

Comment:
That sounds like a good practice. I committed to my clone: https://github.com/geoffviola/ament_package/commit/fc4bc0380f9358b46ebce63a08c5e1288db1a245. Should I create a new PR?